charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -17,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 27.1.0
+version: 27.1.1
```

Comment:
I woluld go for 27.2.0 as the Grafana update seems to be a feature update too.

Reply:
done

Reply:
:-( i updated this via the UI, and it broke the commit signoff,
is there a easy solution to amend a old commit?